database/consts.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Dict
+
+OBJECT_TYPE = Any
+QUERY_TYPE = Dict[str, OBJECT_TYPE]
+COLLECTION_TYPE = Any
```

Comment:
Creating a type specifically for db query response is unnecessary; depending on the query the return type is different. We can copy the response type from the library.

<img width="667" alt="image" src="https://github.com/hsdevelops/cron-telebot/assets/25633486/c062dff6-cc86-4277-971c-1d76f84976ad">